jobsdb/jobsdb.go

```diff
@@ -138,7 +138,8 @@ const (
 )
 
 var globalDBHandle *sql.DB
-var masterBackupEnabled, instanceBackupEnabled, instanceBackupFailedAndAborted bool
+var masterBackupEnabled, instanceBackupFailedAndAborted = new(bool), new(bool)
+var instanceBackupEnabled = make(map[string]*bool)
```

Comment:
Instead of one global `map` variable holding all possible table_prefixes, it would be simpler & more prudent (_abiding to rule no.1: never use global variables!_) to use a single `bool` struct field, and initialise it during `getBackUpSettings`. Same should apply for other relevant configuration options:

- `masterBackupEnabled`
- `instanceBackupFailedAndAborted`

Reply:
Done